Update pstn-usage-report.md (#3443)
MicrosoftDocs/OfficeDocs-SkypeForBusiness: Teams/teams-analytics-and-reports/pstn-usage-report.md

|**1**   |The report can be viewed for trends over the last 7 days or 28 days. |
|**2**   |Each report has a date for when it was generated. The reports usually reflect a 24 to 48 hour latency from time of activity. |
|**3**   |The X axis is the selected date range for the specific report. The Y axis is the total number of calls over the selected time period.<br>Hover over the dot on a given date to see the total calls on that date.  |

> |**4**   |The table gives you a breakdown of PSTN usage per call. <ul><li>**Time stamp (UTC)** is the time the call started.</li><li>**Display name** is the display name of the user. You can click the display name to go to the user's setting page in the Microsoft Teams admin center. The name also can be name of a bot, for example the Call Queue or Cloud Auto Attendant </li><li>**SIP address** is the SIP address of the user or a bot who received or made the call.</li><li>**Caller number** is the number of the user or the bot who made the call. </li>><li>**Callee number** is the number of the user or the bot who recieved the call. On inbound to a Teams user call it will be the Teams user, on outbound from Teams user call it will be the PSTN User </li><li>**Call type** is whether the call was a PSTN outbound or inbound call and the type of call such as a call placed by a user or an audio conference. The calls types you may see include:<br><br>**Teams user call types**<ul><li>**dr_in** - the user received an inbound PSTN call</li><li>**dr_out** - the user placed an outbound PSTN call</li><li>**dr_out_user_conf** - the user added a PSTN participant to the call</li><li>**user_out_transfer** - the user transferred the call to a PSTN number</li><li>**dr_out_user_forwarding** - the user forwarded the call to a PSTN number</li><li>**dr_out_user_transfer** - the user transferred the call to a PSTN number</li><li>**dr_emergency_out** - the user makes an emergency call</li></ul><br>**Teams bots call types**<ul><li>**dr_in_ucap** - an inbound PSTN call to a Teams bot such as auto attendant or call queue</li><li>**dr_out_ucap** - an outbound PSTN call from a Teams bot such as auto attendant or call queue</li></ul><br><li>**Called to** is the number of the user who received the call.</li><li>**Start time (UTC)** Time when the SIP proxy received the final answer (SIP Message  “200 OK”) from the SBC on outbound (Teams/Bot to a PSTN User) or after the SIP Proxy send the Invite to the next hop within Teams backend on inbound call (PSTN User to a Teams/Bot) </li><li>**Invite time (UTC)** When the initial Invite send on outbound from Teams user or bot call to the SBC or received on inbound to Teams or bot call by the SIP Proxy component of Direct Routing from the SBC.</li><li>**Failure time (UTC)** is the time the call failed. For failed calls only.Final SIP Code, Final Microsoft Subcode and Final SIP Phrase provide the reasons why call failed and can help with troubleshooting. </li><li>**End time (UTC)** is the time the call ended. (For successful calls only.)</li><li>**Duration** is how long the call was connected.</li><li>**Number type** is the user's phone number type, such as a service of toll-free number. </li><li>**Media bypass** indicates whether the trunk was enabled for media bypass </li> <li>**SBC FQDN** is the fully qualified domain name (FQDN) of the Session Border Controller (SBC). </li><li>**Azure region for Media** The datacenter that used for media path in non-bypass call. </li><li>**Azure region for Signaling** The datacenter that used for signaling for both bypass and non-bypass calls </li><li>**Event type** is the event type for the call. You'll see Success for successful calls and Attempt for failed calls. </li><li>**Final SIP code** is the code with which the call ended.</li><li>**Final Microsoft subcode** is a code that indicates specific actions that occurred.</li><li>**Final SIP phrase** is the description of the SIP code and Microsoft subcode.</li><li>**Coorelation ID** is a unique identifier for the call that you can use when calling Microsoft Support.</li><li>**Shared Coorelation ID** This field is only visible in the downloadable CSV file and does not exist in the portal. The shared correlation ID exists in at least two calls which are related. Please see detailed description below.</li></ul> To see the information that you want in the table, make sure to add the columns to the table.|

Reviewer: ```suggestion
|**4**   |The table gives you a breakdown of PSTN usage per call. <ul><li>**Time stamp (UTC)** is the time the call started.</li><li>**Display name** is the display name of the user. You can click the display name to go to the user's settings page in the Microsoft Teams admin center. The name can also be the name of a bot, for example the Call Queue or Cloud Auto Attendant. </li><li>**SIP address** is the SIP address of the user or a bot who received or made the call.</li><li>**Caller number** is the number of the user or the bot who made the call. </li>><li>**Callee number** is the number of the user or the bot who received the call. On inbound to a Teams user call it will be the Teams user, on outbound from Teams user call it will be the PSTN User </li><li>**Call type** is whether the call was a PSTN outbound or inbound call and the type of call such as a call placed by a user or an audio conference. The call types you may see include:<br><br>**Teams user call types**<ul><li>**dr_in** - the user received an inbound PSTN call</li><li>**dr_out** - the user placed an outbound PSTN call</li><li>**dr_out_user_conf** - the user added a PSTN participant to the call</li><li>**user_out_transfer** - the user transferred the call to a PSTN number</li><li>**dr_out_user_forwarding** - the user forwarded the call to a PSTN number</li><li>**dr_out_user_transfer** - the user transferred the call to a PSTN number</li><li>**dr_emergency_out** - the user made an emergency call</li></ul><br>**Teams bots call types**<ul><li>**dr_in_ucap** - an inbound PSTN call to a Teams bot such as auto attendant or call queue</li><li>**dr_out_ucap** - an outbound PSTN call from a Teams bot such as auto attendant or call queue</li></ul><br><li>**Called to** is the number of the user who received the call.</li><li>**Start time (UTC)** Time when the SIP proxy received the final answer (SIP Message  “200 OK”) from the SBC on outbound (Teams/Bot to a PSTN User) or after the SIP Proxy send the Invite to the next hop within Teams backend on an inbound call (PSTN User to a Teams/Bot) </li><li>**Invite time (UTC)** When the initial Invite send on outbound from a Teams user or bot call to the SBC or received on inbound to Teams or bot call by the SIP Proxy component of Direct Routing from the SBC.</li><li>**Failure time (UTC)** is the time the call failed. For failed calls only. Final SIP Code, Final Microsoft Subcode and Final SIP Phrase provide the reasons why the call failed and can help with troubleshooting. </li><li>**End time (UTC)** is the time the call ended (for successful calls only).</li><li>**Duration** is how long the call was connected.</li><li>**Number type** is the user's phone number type, such as a service of toll-free number. </li><li>**Media bypass** indicates whether the trunk was enabled for media bypass </li> <li>**SBC FQDN** is the fully qualified domain name (FQDN) of the Session Border Controller (SBC). </li><li>**Azure region for Media** The data center that is used as media path in a non-bypass call. </li><li>**Azure region for Signaling** The data center that used for signaling for both bypass and non-bypass calls </li><li>**Event type** is the event type of the call. You'll see Success for successful calls and Attempt for failed calls. </li><li>**Final SIP code** is the code with which the call ended.</li><li>**Final Microsoft subcode** is a code that indicates specific actions that occurred.</li><li>**Final SIP phrase** is the description of the SIP code and Microsoft subcode.</li><li>**Correlation ID** is a unique identifier for the call that you can use when calling Microsoft Support.</li><li>**Shared Correlation ID** This field is only visible in the downloadable CSV file and does not exist in the portal. The shared correlation ID exists in at least two calls which are related. Please see detailed description below.</li></ul> To see the information that you want in the table, make sure to add the columns to the table.|
```

> |**4**   |The table gives you a breakdown of PSTN usage per call. <ul><li>**Time stamp (UTC)** is the time the call started.</li><li>**Display name** is the display name of the user. You can click the display name to go to the user's settings page in the Microsoft Teams admin center. The name can also be the name of a bot, for example the Call Queue or Cloud Auto Attendant. </li><li>**SIP address** is the SIP address of the user or a bot who received or made the call.</li><li>**Caller number** is the number of the user or the bot who made the call. </li>><li>**Callee number** is the number of the user or the bot who received the call. On an inbound call to a Teams user it will be the Teams user, on an outbound call from a Teams user it will be the PSTN User </li><li>**Call type** is whether the call was a PSTN outbound or inbound call and the type of call such as a call placed by a user or an audio conference. The call types you may see include:<br><br>**Teams user call types**<ul><li>**dr_in** - the user received an inbound PSTN call</li><li>**dr_out** - the user placed an outbound PSTN call</li><li>**dr_out_user_conf** - the user added a PSTN participant to the call</li><li>**user_out_transfer** - the user transferred the call to a PSTN number</li><li>**dr_out_user_forwarding** - the user forwarded the call to a PSTN number</li><li>**dr_out_user_transfer** - the user transferred the call to a PSTN number</li><li>**dr_emergency_out** - the user made an emergency call</li></ul><br>**Teams bots call types**<ul><li>**dr_in_ucap** - an inbound PSTN call to a Teams bot such as auto attendant or call queue</li><li>**dr_out_ucap** - an outbound PSTN call from a Teams bot such as auto attendant or call queue</li></ul><br><li>**Called to** is the number of the user who received the call.</li><li>**Start time (UTC)** is the time when the SIP proxy received the final answer (SIP Message  “200 OK”) from the SBC on an outbound call (Teams/Bot to a PSTN User), or after the SIP Proxy send the Invite to the next hop within the Teams backend on an inbound call (PSTN User to a Teams/Bot) </li><li>**Invite time (UTC)** is the time when the initial Invite was sent on an outbound call from a Teams user or bot call to the SBC, or received on an inbound call to a Teams or bot call by the SIP Proxy component of Direct Routing from the SBC.</li><li>**Failure time (UTC)** is the time the call failed. For failed calls only. Final SIP Code, Final Microsoft Subcode, and Final SIP Phrase provide the reasons why the call failed and can help with troubleshooting. </li><li>**End time (UTC)** is the time the call ended (for successful calls only).</li><li>**Duration** is how long the call was connected.</li><li>**Number type** is the user's phone number type, such as a service of toll-free number. </li><li>**Media bypass** indicates whether the trunk was enabled for media bypass </li> <li>**SBC FQDN** is the fully qualified domain name (FQDN) of the Session Border Controller (SBC). </li><li>**Azure region for Media** is the data center that was used as media path in a non-bypass call. </li><li>**Azure region for Signaling** is the data center that was used for signaling for both bypass and non-bypass calls </li><li>**Event type** is the event type of the call. You'll see Success for successful calls and Attempt for failed calls. </li><li>**Final SIP code** is the code with which the call ended.</li><li>**Final Microsoft subcode** is a code that indicates specific actions that occurred.</li><li>**Final SIP phrase** is the description of the SIP code and Microsoft subcode.</li><li>**Correlation ID** is a unique identifier for the call that you can use when calling Microsoft Support.</li><li>**Shared Correlation ID** is only visible in the downloadable CSV file and does not exist in the portal. The shared correlation ID exists in at least two calls which are related. Please see detailed description below.</li></ul> To see the information that you want in the table, make sure to add the columns to the table.|

Reviewer: ```suggestion
|**4**   |The table gives you a breakdown of PSTN usage per call. <ul><li>**Time stamp (UTC)** is the time the call started.</li><li>**Display name** is the display name of the user. You can click the display name to go to the user's settings page in the Microsoft Teams admin center. The name can also be the name of a bot, for example the Call Queue or Cloud Auto Attendant. </li><li>**SIP address** is the SIP address of the user or a bot who received or made the call.</li><li>**Caller number** is the number of the user or the bot who made the call. </li>><li>**Callee number** is the number of the user or the bot who received the call. On an inbound call to a Teams user it will be the Teams user, on an outbound call from a Teams user it will be the PSTN User. </li><li>**Call type** is whether the call was a PSTN outbound or inbound call and the type of call such as a call placed by a user or an audio conference. The call types you may see include:<br><br>**Teams user call types**<ul><li>**dr_in** - the user received an inbound PSTN call</li><li>**dr_out** - the user placed an outbound PSTN call</li><li>**dr_out_user_conf** - the user added a PSTN participant to the call</li><li>**user_out_transfer** - the user transferred the call to a PSTN number</li><li>**dr_out_user_forwarding** - the user forwarded the call to a PSTN number</li><li>**dr_out_user_transfer** - the user transferred the call to a PSTN number</li><li>**dr_emergency_out** - the user made an emergency call</li></ul><br>**Teams bots call types**<ul><li>**dr_in_ucap** - an inbound PSTN call to a Teams bot such as auto attendant or call queue</li><li>**dr_out_ucap** - an outbound PSTN call from a Teams bot such as auto attendant or call queue</li></ul><br><li>**Called to** is the number of the user who received the call.</li><li>**Start time (UTC)** is the time when the SIP proxy received the final answer (SIP Message  “200 OK”) from the SBC on an outbound call (Teams/Bot to a PSTN User), or after the SIP Proxy send the Invite to the next hop within the Teams backend on an inbound call (PSTN User to a Teams/Bot). </li><li>**Invite time (UTC)** is the time when the initial Invite was sent on an outbound call from a Teams user or bot call to the SBC, or received on an inbound call to a Teams or bot call by the SIP Proxy component of Direct Routing from the SBC.</li><li>**Failure time (UTC)** is the time the call failed. For failed calls only. Final SIP Code, Final Microsoft Subcode, and Final SIP Phrase provide the reasons why the call failed and can help with troubleshooting. </li><li>**End time (UTC)** is the time the call ended (for successful calls only).</li><li>**Duration** is how long the call was connected.</li><li>**Number type** is the user's phone number type, such as a service of toll-free number. </li><li>**Media bypass** indicates whether the trunk was enabled for media bypass. </li> <li>**SBC FQDN** is the fully qualified domain name (FQDN) of the Session Border Controller (SBC). </li><li>**Azure region for Media** is the data center that was used as media path in a non-bypass call. </li><li>**Azure region for Signaling** is the data center that was used for signaling for both bypass and non-bypass calls. </li><li>**Event type** is the event type of the call. You'll see Success for successful calls and Attempt for failed calls. </li><li>**Final SIP code** is the code with which the call ended.</li><li>**Final Microsoft subcode** is a code that indicates specific actions that occurred.</li><li>**Final SIP phrase** is the description of the SIP code and Microsoft subcode.</li><li>**Correlation ID** is a unique identifier for the call that you can use when calling Microsoft Support.</li><li>**Shared Correlation ID** is only visible in the downloadable CSV file and does not exist in the portal. The shared correlation ID exists in at least two calls which are related. Please see detailed description below.</li></ul> To see the information that you want in the table, make sure to add the columns to the table.|
```

|**5**   |Select **Edit columns** to add or remove columns in the table. |
|**6**   |Select **Full screen** to view the report in full screen mode. |

> |**7**   |Select **Export to Excel** to download the data in a comma separated file (CSV) for offline analysis or using as input for your billing system. |

Reviewer: ```suggestion
|**7**   |Select **Export to Excel** to download the data in a comma separated file (CSV) for offline analysis or to use it as input for your billing system. |
```

**Caller/Callee fields conciderations**

> Depends on the call direction the Caller or Calleee names can containg non-E164 numbers.

Reviewer: ```suggestion
Depending on the call direction, the Caller or Callee names can contain non-E164 numbers.
```

> These fields can come form the custoemer SBC(s). There are three formats we see the SBC can send to Direct Routing: E.164, non-E.164 numbers and strings.

Reviewer: ```suggestion
These fields can come form the customer SBC(s). There are three formats that the SBC can send to Direct Routing: E.164 numbers, non-E.164 numbers, and strings.
```

> 1. E.164 proper phone number from a user who has E.164 number to the user who also has it. 

Reviewer: ```suggestion
1. E.164 phone number from a user who has an E.164 number to a user who also has an E.164 number. 
```

> 2. Call from a non-E.164 number. A user from a third-party PBX interconnected with DR makes a call to  Teams user. In this case the caller number might be any non-E.164 number. For example,  +1001 . 

Reviewer: ```suggestion
2. Call from a non-E.164 number. A user from a third-party PBX interconnected with DR makes a call to a Teams user. In this case the caller number might be any non-E.164 number, for example +1001. 
```

> 3. A spammer calls and doesn’t present the number, only the name. For example “Internal Revenue Service”. This string will be shown in the reports

Reviewer: ```suggestion
3. A spammer calls and doesn’t present a number, only a name, for example “Internal Revenue Service”. This string will be shown in the reports.
```

**About Shared Correlation ID**

> The Shared Correlated ID only exists in downloadable Excel and indicates that two or more calls are related. 

Reviewer: ```suggestion
The Shared Correlation ID only exists in downloadable Excel and indicates that two or more calls are related. 
```

> Below is the table that explains different scenarios and when Shared Correlation ID is present

Reviewer: ```suggestion
The following table explains the different scenarios, and when Shared Correlation ID is present
```
